MultisigIntegrationTests/SafeTransactionServiceIntegrationTests.swift

```diff
@@ -25,8 +25,10 @@ class SafeTransactionServiceIntegrationTests: XCTestCase {
     func test_safeInfo_whenSendingNotASafe_returns404Error() {
         let semaphore = DispatchSemaphore(value: 0)
         DispatchQueue.global().async {
-            try? XCTAssertThrowsError(try self.service.safeInfo(at: "0xc778417E063141139Fce010982780140Aa0cD5Ab")) {
-                guard case HTTPClient.Error.entityNotFound(_, _, _) = $0 else {
+            do {
+                _ = try self.service.safeInfo(at: "0xc778417E063141139Fce010982780140Aa0cD5Ab")
```

Comment:
previous implementation produced warnings due to different closure types (`@convention(block)` syntax was conflicting with the async closure's syntax)